src/SFA.DAS.AssessorService.Web/Startup.cs

```diff
@@ -229,7 +200,7 @@ private IServiceProvider ConfigureIoc(IServiceCollection services)
             {
                 config.Scan(_ =>
                 {
-                    _.AssemblyContainingType(typeof(Startup));
+                    _.AssembliesFromApplicationBaseDirectory(c => c.FullName.StartsWith("SFA"));
```

Comment:
@ChrisJWoodcock is this registration any services which the namespace starts with SFA?

Reply:
Yes - this is the same as used in the Api and External API already